samples/dialogs/src/main/java/com/badoo/ribs/samples/dialogs/rib/dialogs_example/DialogsPresenter.kt

```diff
@@ -19,11 +19,13 @@ import com.badoo.ribs.samples.dialogs.rib.dialogs_example.DialogsView.Event.Show
 import com.badoo.ribs.samples.dialogs.rib.dummy.Dummy
 import io.reactivex.Observable.wrap
 import io.reactivex.disposables.CompositeDisposable
-import io.reactivex.functions.Consumer
 
 interface DialogsPresenter {
 
-    fun handle(event: DialogsView.Event)
+    fun handleThemedDialog(event: ShowThemedDialogClicked)
+    fun handleSimpleDialog(event: ShowSimpleDialogClicked)
+    fun handleLazyDialog(event: ShowLazyDialogClicked)
+    fun handleRibDialog(event: ShowRibDialogClicked)
```

Comment:
I think we don't need the event param at all:

```suggestion
    fun handleThemedDialog()
    fun handleSimpleDialog()
    fun handleLazyDialog()
    fun handleRibDialog()
```